src/draft-ietf-rpp-requirements.md

```diff
@@ -76,28 +76,29 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT","SHOULD", "SH
 
 # General
 
-**R1.1.** A well defined architecture MUST be defined for RPP, including a description of the responsibilities of the definded protocol layers. 
+**R1.1.** An architecture MUST be defined for RPP, including a description of the responsibilities of the definded protocol layers. 
+<!-- "well-defined" is subjective -->
 
-**R1.2.** RPP MUST provide a clear, clean, easy to use and self-explanatory interface that can easily be integrated into existing software systems.
-<!-- A> //TODO: [Issue #3](https://github.com/ietf-wg-rpp/rpp-requirements/issues/3) -->
+**R1.2** *REMOVED*
+<!-- R1.2 is way to vague and subjective -->
```

Comment:
This point was already open for discussion and one argument was that such meta-requirement is needed as it was one of the key motivations behind RPP
I will reopen https://github.com/ietf-wg-rpp/rpp-requirements/issues/3 - glad to continue discussion there (not in the PR).